packages/jobs/lib/crons/autoIdleDemo.ts

```diff
@@ -0,0 +1,128 @@
+import { schedule } from 'node-cron';
+import {
+    CommandToActivityLog,
+    ErrorSourceEnum,
+    ScheduleStatus,
+    SyncClient,
+    SyncCommand,
+    SyncConfig,
+    createActivityLog,
+    createActivityLogMessageAndEnd,
+    db,
+    errorManager,
+    updateSuccess as updateSuccessActivityLog,
+    updateScheduleStatus,
+    isErr
+} from '@nangohq/shared';
+import tracer from 'dd-trace';
+
+export async function cronAutoIdleDemo(): Promise<void> {
+    schedule('1 * * * *', () => {
+        const span = tracer.startSpan('cron.syncs.idleDemo');
```

Comment:
not directly related to this PR but a meta comment that we're getting a lot of different span names and not sure if I even know all of them. It might be nice to have a centralized place to reference all the span names we have.

Reply:
Good call, that would help with discoverability for sure

Reply:
fixed in [e6c2681](https://github.com/NangoHQ/nango/pull/1631/commits/e6c2681df513185d4e4f39587c7afb7980aa8557)